doc/source/device-overview.rst

```diff
@@ -34,15 +36,52 @@ Under the hood, `Device` uses a metaclass to allow for
         load_cmd = Cpt(EpicsSignal, 'Cmd:Load-Cmd.PROC')
         unload_cmd = Cpt(EpicsSignal, 'Cmd:Unload-Cmd.PROC')
         execute_cmd = Cpt(EpicsSignal, 'Cmd:Exec-Cmd')
+
         status = Cpt(EpicsSignalRO, 'Sts-Sts')
 
-    my_robot = Robot('pv_prefix:', name='my_robot')
+    my_robot = Robot('PV_PREFIX:', name='my_robot',
+                     read_attrs=['sample_number', 'status'])
+
+Which creates an instance ``my_robot`` with 5 children
+
+   ======================   ===============================    =====================
+   python attribute         PV name                            in ``read()``
+   ======================   ===============================    =====================
+   my_robot.sample_number   'PV_PREFIX:ID:Tgt-SP'              Y
+   my_robot.load_cmd        'PV_PREFIX:CMD:Load-Cmd.PROC'      N
+   my_robot.unload_cmd      'PV_PREFIX:CMD:Unload-Cmd.PROC'    N
+   my_robot.execute_cmd     'PV_PREFIX:CMD:Exec-Cmd'           N
+   my_robot.status          'PV_PREFIX:Sts-Sts'                Y
+   ======================   ===============================    =====================
+
+only 2 of which will be included when reading from the robot.  You could now use this
+device in a scan like
+
+.. code-block:: python
+
+   import bluesky.plans as bp
+
+   def load_sample(robot, sample):
+       yield from bp.mv(robot.sample_number, sample)
+       yield from bp.mv(robot.load_cmd, 1)
+       yield from bp.mv(robot.execute_cmd, 1)
+
+   def unload_sample(robot):
+       yield from bp.mv(robot.unload_cmd, 1)
+       yield from bp.mv(robot.execute_cmd, 1)
+
+   def robot_plan(list_of_samples):
+       for sample in list_of_samples:
+           # load the sample
+	   yield from load_sample(my_robot, sample)
```

Comment:
The robot is actually probably not a good first example of a device. This skips a couple subtle steps (define `load_sample` plan, register `load_sample` message with RunEngine).